keycloak/keycloak-operator
# Intly 3952- auto configure browser identity provider redirector

## JIRA ID

INTLY-3952

## Additional Information

Implement the automatic configuration of the browser authentication flow. This is a convenience feature that does not seem to be possible to achieve using realm properties.

## Verification Steps

1. Deploy keycloak
2. Create the `realm-with-users` example realm
3. It should succeed.
4. Login to keycloak as admin
5. Go to: User_realm -> Authentication
6. Make sure the Flows tab is selected
7. From the dropdown, choose Browser
8. In the table there should be a row with ` Identity Provider Redirector(keycloak-operator-browser-redirector) | Alternative`
9. In this row, click on Actions -> Configure
10. The default identity provider should be set to github

## Checklist:
